src/Raven.Studio/typescript/models/database/settings/databaseSettingsModels.ts

```diff
@@ -6,6 +6,11 @@ import activeDatabaseTracker = require("common/shell/activeDatabaseTracker");
 
 type configurationOrigin = "Default" | "Server" | "Database";
 
+function findMatchingKey<T>(availableKeys: string[], values: T): string | undefined {
```

Comment:
The generic does nothing in this case. It is better to just specify the type of values

```suggestion
function findMatchingKey(availableKeys: string[], values: Record<string, Raven.Server.Config.ConfigurationEntrySingleValue>): string | undefined {
```